context/llpcCompiler.cpp

```diff
@@ -128,10 +127,7 @@ opt<uint32_t> ShadowDescTablePtrHigh("shadow-desc-table-ptr-high",
                                      desc("High part of VA for shadow descriptor table pointer"),
                                      init(2));
 
-// -enable-dynamic-loop-unroll: Enable dynamic loop unroll.
-opt<bool> EnableDynamicLoopUnroll("enable-dynamic-loop-unroll", desc("Enable dynamic loop unroll"), init(false));
```

Comment:
Please bump LLPC version, and keep the definition of this option in old version. We need keep LLPC code compatible with old xgl code. thanks!